### ISSUE TITLE
Icons still using OS X 10.9 “Mavericks” style + partial solution

### ISSUE THREAD

**Issue by joaofrgomes** [MaddTheSane/Boxer#50]
Hi! I've been using Boxer for quite a while, and it always bugged me, ever since Apple came up with these newfangled icons, that Boxer didn't adhere to them.

I have already made a dark version for the Boxer games folder, and can make a light one to go with it if you want. Check it out!
[Attachment deleted, consolidated package available below]

**Comment by MaddTheSane** [comment]
Looks good.

**Comment by joaofrgomes** [comment]
Good to know… I have more! It's not yet the light version of the folder (I'll get to that one later), but in this case, an updated DOS app icon to match macOS Big Sur's (and later) Terminal icon, complete with the missing 512x512@2x and 32x32@2x versions (the latter of which also with added scanlines for good measure, of course ;) ).
[DOSapp.icns.zip](https://github.com/MaddTheSane/Boxer/files/10788002/DOSapp.icns.zip)

**Comment by joaofrgomes** [comment]
Aaaaand the light version is done!

I'm reuploading both icons here as a package to avoid any confusion. Enjoy! ;)

[gamefolder.zip](https://github.com/MaddTheSane/Boxer/files/10788316/gamefolder.zip)

**Comment by joaofrgomes** [comment]
Sorry, I accidentally closed the thread.

There are more icons that I wouldn't mind helping with, so I'll leave it open (unless you think it warrants opening a new issue, that is).

**Comment by joaofrgomes** [comment]
And here's yet another proposal, this time for the main icon. Staying true to its long legacy, I tried reducing the current freestanding icon to 75% of its size (which works out pretty nicely as far as icon sizes go, seeing how they are multiples of 8 px.) and superimposing it to the “DOSapp” command prompt icon I've just posted (with mild changes to the 16x16 version, so that the “c:” element isn't obscured too much):

[boxer.icns.zip](https://github.com/MaddTheSane/Boxer/files/10788814/boxer.icns.zip)

I'm guessing that you may want to go for a flatter approach in the future but, in a pinch, it can work nicely with Big Sur and above, as it follows the “roundrect background + 3D tool” convention Apple uses in the likes of TextEdit, Script Editor, Digital Color Meter, etc.

**Comment by MaddTheSane** [comment]
Agreed, a flatter look on the gloves would fit the look of the current macOS better.

**Comment by joaofrgomes** [comment]
Indeed. At the moment I don’t exactly have the wherewithal

**Comment by joaofrgomes** [comment]
Sorry, closed the thread again by accident (writing on Safari for iOS is even worse, yikes!).

At this point I don’t exactly have the wherewithal to help with such an extensive redesign (which might even extend to other parts of the app, though I feel some skeumorphism, such as in OpenEmu, is appropriate for such a retro-minded app, especially when it comes to that lovely Roland MT-32 artwork, the floppy and jewel case icons – which don’t have 512x512@2x variants, you might want to consider contacting the original artist to ask them for the master vector files –, etc.), but I can always provide whoever volunteers with my master Affinity Designer and Photoshop files and let them take it from there.

Or, who knows, maybe I can help in time for a major new release, but I’m a bit tied up with my PhD for the next 1–2 years. 😬
